resources/js/Pages/Results.vue

```diff
@@ -177,6 +214,12 @@
             }
         },
         methods: {
+            showInstructionsDialog(e: Event) {
+                e.preventDefault();
```

Comment:
We probably don't need this anymore, since we are not using a link.

Reply:
done